rmf_traffic/test/unit/agv/test_Planner.cpp

```diff
@@ -458,6 +458,159 @@ SCENARIO("Test Options", "[options]")
     CHECK(*default_options.interrupt_flag());
   }
 
+  Planner::Options default_maxcostestimate(nullptr);
+  WHEN("Maximum cost estimate is default after default construction")
+  {
+    // Converting an optional to a bool returns false if it's empty, which is
+    // the default for constructing Options
+    CHECK(!default_maxcostestimate.maximum_cost_estimate());
```

Comment:
```suggestion
    CHECK_FALSE(default_max_cost_estimate_options.maximum_cost_estimate());
```

Catch2 provides `CHECK_FALSE` which is the recommended way to test for false expectations. It makes the readout more clear when the test fails, and I think it makes the test expectations more explicit.